### ISSUE TITLE
Does not work anymore ,Upload errors.

### ISSUE THREAD

**Issue by b1tw0rker** [trippo/ResponsiveFilemanager#725]
I just ask myself, if this tools still is up to date and supports pHP 8. I get these error:
PHP Warning:  Trying to access array offset on value of type null in /home/httpd/x9ksd9/htdocs/filemanager/UploadHandler.php on line 496PHP message: PHP Warning:  Trying to access array offset on value of type null in /home/httpd/x9ksd9/htdocs/filemanager/UploadHandler.php on line 1429

On screen these error:
saveImage: This is not a resource.

It look slike the tool is not supported anymore.

**Comment by jshster** [comment]
If you download the latest build you will find these errors will be gone.  The project is moving, albeit a little covertly!

**Comment by dleffler** [comment]
Yes, download the code, NOT the latest release to fix issues with PHP x8.x
